### ISSUE TITLE
Get-Secret can't get a secret from a folder (subgroup).

### ISSUE THREAD

**Issue by l2aggron** [JustinGrote/SecretManagement.KeePass#56]
Hello!
It looks like you've already worked on a related problem:
https://github.com/JustinGrote/SecretManagement.KeePass/issues/10
I am trying to access a secret in a folder but without success, is it not implemented or am I doing something wrong?
My setup: Win10x64, PS 5.1, database format is KBDX 4.0, created in KeePassXC-2.6.4-Win64.
![image](https://user-images.githubusercontent.com/25399254/120892373-a97c2e00-c616-11eb-928c-607e26b227e5.png)

**Comment by l2aggron** [comment]
It looks like the problem in compatibility with the database format.
I created a new DB using Register-KeepassSecretVault and I can properly access the secrets in the folders.
I checked the database, it has the format 3.1.
When I created the db using KeePassXC, selecting this format, and registered the storage with the ShowFullTitle and ShowRecycleBin parameters, I can see the secrets, but I can't see the folders and trash.
Similar behavior when creating db in the latest version of KeePass (2.48.1).

P.S. I would be happy if you would add documentation on the compatibility of the module and database formats, or update the necessary components to work with the latest versions of database formats.
P.P.S. Anyway, thank you for your work!
